libdrgn/debug_info.c

```diff
@@ -336,11 +336,13 @@ drgn_debug_info_report_error(struct drgn_debug_info_load_state *load,
 	}
 	if (load->num_errors == 0 &&
 	    !string_builder_append(&load->errors,
-				   "could not get debugging information for:"))
+				   "Missing some debuginfo (see https://drgn.readthedocs.io/en/latest/getting_debugging_symbols.html):"))
```

Comment:
Nitpick: we might as well be consistent in terminology between this error message and the documentation page.
Super nitpick: our error messages are typically not capitalized.
```suggestion
				   "missing some debugging symbols (see https://drgn.readthedocs.io/en/latest/getting_debugging_symbols.html):"))
```